app/lib/actions/action_with_sub_plans.rb

```diff
@@ -3,6 +3,7 @@ class Actions::ActionWithSubPlans < Actions::EntryAction
     middleware.use Actions::Middleware::KeepCurrentUser
 
     include Dynflow::Action::WithSubPlans
+    include Dynflow::Action::WithBulkSubPlans
```

Comment:
I don't think we should include this into the `ActionWithSubPlans` but rather into `BulkAction` and `RunHostsJob` action: the action with sub plans doesn't have to be bulk at the end